pkg/operator/staticpod/controller/backingresource/backing_resource_controller_test.go

```diff
@@ -54,6 +54,11 @@ func TestBackingResourceController(t *testing.T) {
 					ManagementState: operatorv1.Managed,
 				},
 				&operatorv1.OperatorStatus{},
+				&operatorv1.StaticPodOperatorSpec{
```

Comment:
no.  the backing resource controller should take a normal operator client

Reply:
was that something that was overlooked with this change? https://github.com/openshift/library-go/pull/181/files#diff-490940f3a00b192c5a924281217a65d5R55